### PR TITLE
Add net_vlan platform agnostic module

### PR DESCRIPTION
**Pull request by rcarrillocruz**

### CODE REVIEW

File: lib/ansible/modules/network/net_vlan.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  state:
+    description:
+      - State of the VLAN configuration. 
+    default: present
+    choices: ['present', 'absent', 'active', 'suspend']
+"""
+
+EXAMPLES = """
+- name: configure hostname and domain name
+  net_system:
```
Comment (ganeshrn) [inline lib/ansible/modules/network/net_vlan.py]: Should the example be for net_vlan?

File: lib/ansible/modules/network/net_vlan.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                    'status': ['preview'],
+                    'supported_by': 'core'}
+
+
+DOCUMENTATION = """
+---
+module: net_vlan
+version_added: "2.4"
+author: "Ricardo Carrillo Cruz (@rcarrillocruz)"
+short_description: Manage VLANs on network devices
```
Comment (gundalow) [inline lib/ansible/modules/network/net_vlan.py:32]: `short_description: Manage VLANs on VyOS devices`

Comment (rcarrillocruz) [reply lib/ansible/modules/network/net_vlan.py:32]: I WIP'd it, but in this case no, this is net_vlan therefore no VyOS, but network devices in general

Comment (gundalow) [reply lib/ansible/modules/network/net_vlan.py:32]: Ignore this, wrong PR

**Review by gundalow — APPROVED** [review]

**Pull request merged**